.github/workflows/qgis.yaml

```diff
@@ -16,11 +16,9 @@ jobs:
       matrix:
         version:
           # Old LTR
-          - '3.4'
-          # LTR
           - '3.10'
-          # Last
-          - '3.14'
+          # Last LTR
+          - '3.16'
```

Comment:
The last LTR is still the 3.10 no ?
3.16 is only a standard release

Reply:
@ger-benjamin : see https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule

Reply:
Thanks you are right.
They are not up to date on their whole website. Sometimes (like on the download page or simply the flag at the upper-left corner) they still say the 3.10 is the latest LTR.

Reply:
@ger-benjamin strictly your right that the version 3.16 is the last LTR only from 2021-02-19...